great-idea/css/index.css

```diff
@@ -63,4 +63,252 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Copy and paste your work from yesterday here and start to refactor into flexbox */
\ No newline at end of file
+/* Copy and paste your work from yesterday here and start to refactor into flexbox */
+
+.container {
+	margin: 0 auto;
+	width: 880px;	
+}
+  
+/*****header*****/
+	
+.nav-main {
+	display: flex;
+	width: 800px;
+	justify-content: space-evenly;
+}
+
+.nav-main-page {
+	display: flex;
+	align-items: center;
+	justify-content: space-evenly;
+}
+/* .nav-main ul {
+	justify-content: space-evenly;
+} */
+
+.logo {
+	/* display: inline-block; */
+}
+
+header nav a {
+/* display: inline-block; */
+margin-right: 65px;
+font-size: 14px;
+color: gray;
+text-decoration: none;
+}
+
+header nav a:hover {
+text-decoration: underline;
+}
+
+/* header nav a:first-child {
+margin-left: 0;
+}
+
+header nav a:last-child {
+margin-right: 0;
+} */
+
+.services-header-img {
+	padding-top: 30px;
+	padding-bottom: 20px;
+}
+
+/* cta */
+
+.cta {
+	display: flex;
+}
+
+.cta .cta-text {
+/* display: inline-block; */
+/* vertical-align: top; */
+font-size: 77px;
+margin-top: 120px;
+text-align: center;
+padding: 0px 75px;
+}
+
+.cta .cta-text button {
+background: white;
+vertical-align: top;
+margin-top: 6px;
+padding: 8px 44px;
+font-size: 17px;
+border: 1px solid black;
+cursor: pointer;
+}
+
+.cta .cta-text button:hover {
+background: black;
+color: white;
+}
+
+.cta img {
+/* display: inline-block; */
+margin-top: 69px;
+}
+
+.main-content {
+	display: flex;
+border-top: 1px solid black;
+margin-top: 76px;
+padding-top: 39px;
+flex-wrap: wrap;
+justify-content: center;
+}
+
+
+.top-content {
+	display: flex;
+}
+
+.bottom-content {
+
+}
+/* .main-content .top-content {
+padding: 0 10px;
+
+} */
+
+.main-content .top-content .text-content {
+/* width: 410px;
+display: inline-block; */
+margin-right: 30px;
+}
+
+
+/* .main-content .top-content .text-content:last-child {
+margin-right: 0;
+}
+ */
+
+.main-content .middle-img {
+margin-top: 45px;
+}
+
+.main-content .bottom-content {
+	display: flex;
+margin-top: 37px;
+/* padding: 0 10px;  */
+}
+
+.main-content .bottom-content .text-content {
+/* width: 275px;
+display: inline-block; */
+display: flex;
+flex-wrap: wrap;
+margin-right: 12px;
+}
+
+/* .main-content .bottom-content .text-content:last-child {
+margin-right: 0;
+} */
+
+.contact {
+	display: flex;
+	flex-direction: column;
+margin-top: 57px;
+border-top: 1px solid black;
+padding: 39px 0 0;
+}
+
+.contact p {
+margin: 20px 0;
```

Comment:
The margin here could decrease a little bit. The space between the p elements is a bit large.